mammoth/docx/body_xml.py

```diff
@@ -350,30 +349,50 @@ def break_(element):
             warning = results.warning("Unsupported break type: {0}".format(break_type))
             return _empty_result_with_message(warning)
 
-    
     def inline(element):
+        content_info = {'width': '', 'height': '', 'alt_text': ''}
```

Comment:
A dictionary feels unnecessary here. I'd just use a variable each for width, height, and alt text.

Reply:
These variables are passed around a bit and are generally not needed. (passed through) I was thinking on adding more attributes later. `image_crop` for example is semantical. The idea was to create a little info dictionary with things we know about the image to be used by image filters. So we don't have to change a  bunch of functions that just pass through image info. Said that I agree setting it up like this might be clumsy. And there might be better design patterns?

Reply:
Using dictionaries in this way rather than ordinary variables makes it harder for static analysis tools like linters (and humans) to check that the code is correct. I don't see a problem just using variables as `alt_text` was before.

Reply:
OK cool, will update